game/static/game/css/landing.css

```
@@ -590,6 +551,31 @@ body {
     align-items: center;
 }
 
+@import url('https://fonts.googleapis.com/css2?family=Rajdhani:wght@600&display=swap');
+
+.hero-tagline {
+    font-size: clamp(1.1rem, 2.5vw, 1.4rem);
+    font-weight: 600;
+    color: var(--text-primary);
+    margin: 0 auto 2.8rem auto;
+    letter-spacing: 0.5px;
+    font-family: 'Rajdhani', sans-serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Rajdhani" (font-family-name-quotes)

(font-family-name-quotes)